CHANGELOG.md

```diff
@@ -548,6 +548,7 @@ Released with 1.0.0-beta.37 code base.
 -  Fix static tuple encoding (#4673) (#4884)
 -  Fix bug in handleRevert logic for eth_sendRawTransaction (#4902)
 -  Fix resolve type of getBlock function (#4911)
+-  Fix typos in web3-eth-accounts.rst & TESTING.md (#5047)
```

Comment:
It should be under 1.7.5 as 1.7.4 is already frozen. https://github.com/ChainSafe/web3.js/blob/release/1.7.4/CHANGELOG.md#174

Reply:
@andreistefanwork

Reply:
Correct .. should I add the 1.7.5 entry in `CHANGELOG.md`? (it's not in the file on 1.x currently)